src/AccessibilityInsights/MainWindowHelpers/Initialize.cs

```diff
@@ -114,6 +115,9 @@ private static void PopulateConfigurations()
             // when AccessibilityInsights starts up. 
             ConfigurationManager.GetDefaultInstance().AppConfig.IsUnderElementScope = true;
 
+            // Configure the correct ReleaseChannel for autoupdate
+            Container.SetAutoUpdateReleaseChannel(ConfigurationManager.GetDefaultInstance().AppConfig.ReleaseChannel.ToString());
+
             // enable/disable telemetry
             if (ConfigurationManager.GetDefaultInstance().AppConfig.EnableTelemetry)
                 TelemetryController.EnableTelemetry();
```

Comment:
>TelemetryController [](start = 16, length = 19)

This is where the Container currently gets initialized, so we need to set the ReleaseChannel before reaching this point